.github/workflows/ci.yml

```diff
@@ -22,7 +22,10 @@ jobs:
       run: |
         sudo apt install graphviz
         python -m pip install -U pip
-        pip install tox
+        pip install tox bandit
```

Comment:
This should really be installed inside of tox.  That way it can be installed automatically when people run 'tox' locally, and also the version of it can be pinned and managed same as our other dependencies.

Can you please do it that way?  Add it to test-requirements.in and test-requirements.txt, same as other testing tools like mypy, pytest, pylint.

Reply:
Also, this is what the following messages in the log are referring to.

```
bandit-test run-test: commands[0] | bandit -r -ll -ii .
WARNING: test command found but not installed in testenv
  cmd: /opt/hostedtoolcache/Python/3.9.15/x64/bin/bandit
  env: /home/runner/work/exodus-lambda/exodus-lambda/.tox/bandit-test
Maybe you forgot to specify a dependency? See also the allowlist_externals envconfig setting.

DEPRECATION WARNING: this will be an error in tox 4 and above!
```